tree/dataframe/test/dataframe_snapshot.cxx

```diff
@@ -1008,6 +1008,7 @@ TEST(RDFSnapshotMore, TClonesArray)
 // ROOT-10702
 TEST(RDFSnapshotMore, CompositeTypeWithNameClash)
 {
+   GTEST_SKIP() << "Skipping test with interpreted class without std::type_info.";
```

Comment:
What will happen with this test? Will it be removed?

Reply:
Need to update it to have dictionary info